clients/admin-ui/src/features/data-discovery-and-detection/components/VendorMatchBadge.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Badge,
+  Box,
+  HStack,
+  Image,
+  QuestionIcon,
+  Text,
+  Tooltip,
+} from "fidesui";
+
+interface VendorMatchBadgeProps {
+  vendorName?: string | null;
+  vendorLogoUrl?: string | null;
+  confidence?: "high" | "medium" | "low" | null;
+  isUnknown?: boolean;
+}
+
+export const VendorMatchBadge = ({
+  vendorName,
+  vendorLogoUrl,
+  confidence,
+  isUnknown = false,
+}: VendorMatchBadgeProps) => {
+  // Unknown vendor case
+  if (isUnknown || !vendorName) {
+    return (
+      <Badge colorScheme="gray" variant="outline" fontSize="xs">
+        <HStack spacing={1}>
+          <QuestionIcon boxSize={3} />
+          <Text>Unknown Vendor</Text>
+        </HStack>
+      </Badge>
+    );
+  }
```

Comment:
Can we return early, and isUnknown is redundant. It is just derived from vendorName anyway. 

```suggestion
export const VendorMatchBadge = ({
  vendorName,
  vendorLogoUrl,
  confidence
}: VendorMatchBadgeProps) => {
  if (!!vendorName) return;
  return (
    <Badge colorScheme="gray" variant="outline" fontSize="xs">
      <HStack spacing={1}>
        <QuestionIcon boxSize={3} />
        <Text>Unknown Vendor</Text>
       </HStack>
    </Badge>
  );
```